Makefile

```diff
@@ -1,6 +1,6 @@
 CARGO = cargo
 NEAR  = near
-FEATURES = contract
+FEATURES = contract,log
```

Comment:
This shouldn't be on by default.

Reply:
There was a [discussion about this on Slack](https://near-protocol.slack.com/archives/C020JMC9U15/p1620667119070500?thread_ts=1620661443.048500&cid=C020JMC9U15).

Summary is: if something goes wrong in production there's basically no way to know what happened without logging and it's not that expensive to have it (@mrLSD measured it to be ~0.5 Tgas which is much much less than any operation currently).